  File "pagraph.py", line 115, in run_experiment
    run_experiment_on_graph(graphname, model,  hidden_size, batch_size, cache)
  File "pagraph.py", line 93, in run_experiment_on_graph
    res = start_client(filename, model, hidden_size, batch_size, cache_per)
  File "pagraph.py", line 76, in start_client
    accuracy = float(re.findall("accuracy: (\d+\.\d+)", output)[0])
Traceback (most recent call last):
  File "pagraph.py", line 115, in run_experiment
    run_experiment_on_graph(graphname, model,  hidden_size, batch_size, cache)
  File "pagraph.py", line 93, in run_experiment_on_graph
    res = start_client(filename, model, hidden_size, batch_size, cache_per)
  File "pagraph.py", line 76, in start_client
    accuracy = float(re.findall("accuracy: (\d+\.\d+)", output)[0])
IndexError: list index out of range
  File "pagraph.py", line 115, in run_experiment
    run_experiment_on_graph(graphname, model,  hidden_size, batch_size, cache)
  File "pagraph.py", line 93, in run_experiment_on_graph
    res = start_client(filename, model, hidden_size, batch_size, cache_per)
  File "pagraph.py", line 76, in start_client
    accuracy = float(re.findall("accuracy: (\d+\.\d+)", output)[0])
Traceback (most recent call last):
  File "pagraph.py", line 115, in run_experiment
    run_experiment_on_graph(graphname, model,  hidden_size, batch_size, cache)
  File "pagraph.py", line 93, in run_experiment_on_graph
    res = start_client(filename, model, hidden_size, batch_size, cache_per)
  File "pagraph.py", line 76, in start_client
    accuracy = float(re.findall("accuracy: (\d+\.\d+)", output)[0])
IndexError: list index out of range
  File "pagraph.py", line 115, in run_experiment
    run_experiment_on_graph(graphname, model,  hidden_size, batch_size, cache)
  File "pagraph.py", line 93, in run_experiment_on_graph
    res = start_client(filename, model, hidden_size, batch_size, cache_per)
  File "pagraph.py", line 76, in start_client
    accuracy = float(re.findall("accuracy: (\d+\.\d+)", output)[0])
Traceback (most recent call last):
  File "pagraph.py", line 115, in run_experiment
    run_experiment_on_graph(graphname, model,  hidden_size, batch_size, cache)
  File "pagraph.py", line 93, in run_experiment_on_graph
    res = start_client(filename, model, hidden_size, batch_size, cache_per)
  File "pagraph.py", line 76, in start_client
    accuracy = float(re.findall("accuracy: (\d+\.\d+)", output)[0])
IndexError: list index out of range
